android/src/main/kotlin/com/hiennv/flutter_callkit_incoming/FlutterCallkitIncomingPlugin.kt

```diff
@@ -280,6 +284,39 @@ class FlutterCallkitIncomingPlugin : FlutterPlugin, MethodCallHandler, ActivityA
                     }
                     callkitNotificationManager?.requestNotificationPermission(activity, map)
                 }
+
+                "canUseFullScreenIntent" -> {
+                    val notificationManager: NotificationManager =
+                        context?.getSystemService(Context.NOTIFICATION_SERVICE) as NotificationManager
+
+                    val canUseFullScreenIntent =
+                        Build.VERSION.SDK_INT < Build.VERSION_CODES.UPSIDE_DOWN_CAKE || notificationManager.canUseFullScreenIntent()
+                    result.success(canUseFullScreenIntent)
+                }
+
+                "openFullScreenIntentSettings" -> {
+                    if (Build.VERSION.SDK_INT < Build.VERSION_CODES.UPSIDE_DOWN_CAKE) {
+                        return
+                    }
+                    val packageName = call.arguments as? String ?: ""
```

Comment:
It is preferable to get a package name from `context.packageName`.

MethodCall.arguments is not for getting a package name.
https://api.flutter.dev/javadoc/io/flutter/plugin/common/MethodCall.html#arguments

And I would prefer an early return if a context is null.

Reply:
I fixed it 🙏